examples/fonts.ts

```diff
@@ -0,0 +1,17 @@
+import BlackItalicFont from "./fonts/Inter-BlackItalic.woff2"
+import LightFont from "./fonts/Inter-Thin.woff2"
+
+export const CUSTOM_FONTS = [
+  {
+    family: "Inter",
+    style: "normal",
+    weight: "100",
+    source: `url("${LightFont}") format("woff2")`,
```

Comment:
If we hardcode the location here it saves us from needing to run the fonts through webpack
```suggestion
    source: `url("./fonts/Inter-Thin.woff2") format("woff2")`,
```